internal/pkg/service/stream/storage/level/local/diskreader/volume.go

```diff
@@ -174,7 +176,7 @@ func (v *Volume) OpenReader(slice *model.Slice) (r Reader, err error) {
 	}
 
 	// Open file
-	filePath := slice.LocalStorage.FileName(v.Path(), "my-node")
+	filePath := slice.FileName(v.Path(), "my-node")
```

Comment:
Probably better to pass the `FileName` function instead of whole `LocalStorage` object ?